src/pudl/workspace/datastore.py

```diff
@@ -407,6 +407,7 @@ def get_unique_resource(self, dataset: str, **filters: Any) -> bytes:
 
     def get_zipfile_resource(self, dataset: str, **filters: Any) -> zipfile.ZipFile:
         """Retrieves unique resource and opens it as a ZipFile."""
+        # TODO: add some check if the result of get_unique_resource isn't a zip file.
         return zipfile.ZipFile(io.BytesIO(self.get_unique_resource(dataset, **filters)))
```

Comment:
hey @rousik ! since the excel extractor no longer needs to deal with a single file in the case of eia860m and can always expect a zipped resource, I stripped out that possibility from the excel extractor. But i thought it might be good to have a very clear message to let you know if you somehow are pointing to a non-zipped resource. Do you know of an easy way to check for that? also, this may be a moot question bc maybe the error message that would result from this would be clear enough.